contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--3.1.0--3.2.0.sql

```diff
@@ -6,5 +6,17 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
 DROP CAST IF EXISTS(NUMERIC AS sys.BIT);
 CREATE CAST (NUMERIC AS sys.BIT) WITH FUNCTION sys.numeric_bit (NUMERIC) AS IMPLICIT;
 
+DROP CAST IF EXISTS(sys.VARCHAR AS sys.MONEY);
+
+DROP FUNCTION IF EXISTS sys.varchar2money(sys.VARCHAR) CASCADE;
+
+CREATE OR REPLACE FUNCTION sys.varchar2money(sys.VARCHAR)
```

Comment:
General comment (non-blocking): Fixeddecimal is a supported type in babelfish so it's still possible that some other query returns it and we still can hit the same issue. We are trying to fix a small set of such issues where cast from varchar to money is involved but we should think about supporting this type in TDS sender layer as a complete fix, maybe in future.